angular/projects/admin-nrpti/src/app/mines/mines-rows/mines-table-row.component.html

```diff
@@ -1,8 +1,25 @@
 <td scope="row" data-label="Name" class="col-3">{{ rowData.name || '-' }}</td>
+<td scope="row" data-label="Permit Numbers" class="col-2">
+  <ul *ngIf="rowData.permitNumbers && rowData.permitNumbers.length > 0; else elseBlock">
+    <li *ngFor="let permit of rowData.permitNumbers">{{ permit }}</li>
+  </ul>
+  <ng-template #elseBlock>-</ng-template>
+</td>
+<td scope="row" data-label="Type" class="col-1">{{ rowData.type || '-' }}</td>
 <td scope="row" data-label="Permittee" class="col-2">{{ rowData.permittee || '-' }}</td>
-<td scope="row" data-label="Type" class="col-2">{{ rowData.type || '-' }}</td>
-<td scope="row" data-label="Status" class="col-2">{{ rowData.status || '-' }}</td>
-<td scope="row" data-label="Region" class="col-2">{{ rowData.region || '-' }}</td>
+<td scope="row" data-label="Region" class="col-1">{{ rowData.region && displayRegion(rowData.region) || '-' }}</td>
+
+<td scope="row" data-label="Published State" class="col-2 d-flex">
+  <div *ngIf="isPublic(); then published else unpublished"></div>
```

Comment:
Don't really need a function here. You could directly put in this.rowData.read && this.rowData.read.includes('public');

Reply:
Unless you think something else might use this check

Reply:
Was more just preference, I don't really like cluttering the HTML with a bunch of logic. I don't think it will grow though so I will move it there as it is small.